### DISCUSSION TITLE
How to Get the Req and Res Headers and Body in plain text format from MITMproxy

### DISCUSSION THREAD

**Discussion by chris5593** [mitmproxy/mitmproxy#4990] [General]
How to Get the Req and Res Headers and Body in plain text format from MITMproxy

**Comment by mhils** [comment]
Depends on the interface you are using.

- **mitmweb** has a dedicated button for this.
- **mitmdump -v** includes all headers in the output
- **mitmproxy** has export commands. The shortest way is probably to press `e` in the flow list. Or you open the command prompt (`:`) and then type one of the export commands, e.g. `export.clip raw_request @focus` to send it straight to your clipboard.

**Reply by chris5593** [reply]
So Maxmilian, are these commands documented? I am looking for an in and out documentation on mitmproxy. Please guide me where to find it with all the options and everything explained. I know there is a documentation section in the site but im unable to see these commands like mitmdump -v. May be i havent searched everywhere.But It will be a great help if you can guide me

**Reply by takekazuomi** [reply]
I was looking for the same thing too. This answer is perfect. 
In my impression, the documentation is detailed and excellent. However,  it was hard to find an easy way to use it. 
Thank you for your fantastic project. Very helpful.
